packages/project-utils/bundling/ssr/buildAppSSR.js

```diff
@@ -3,6 +3,7 @@ const fs = require("fs-extra");
 
 module.exports.buildAppSSR = async (
     { app = null, indexHtmlPath = null, createAppPath = null, output, ...options },
+    // eslint-disable-next-line
```

Comment:
I think we can remove this line / the unused `context` below. I don't see it being used anywhere.